src/test/java/apoc/config/ConfigTest.java

```diff
@@ -21,64 +16,23 @@
 public class ConfigTest {
 
     private GraphDatabaseService db;
-    private File testFile;
 
     @Before
     public void setUp() throws Exception {
         db = new TestGraphDatabaseFactory().newImpermanentDatabaseBuilder()
```

Comment:
where did all these tests go?

Reply:
We moved all the tests into the class `FileUtilsTest`